### PR TITLE
Update 04_data-persistence.md

### PR DESCRIPTION
**Pull request by rusty-block-one**
Clarification in step 4 as to where the person struct should be created. Removed unnecessary code from update of addressbook.

### CODE REVIEW

File: docs/02_getting-started/03_smart-contract-development/04_data-persistence.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ class [[eosio::contract("addressbook")]] addressbook : public eosio::contract {
 ```
 
 ## Step 4: Create The Data Structure for the Table
-Before a table can be configured and instantiated, we need to define a struct that represents the data structure of the address book. Since it is an address book, the table will contain people, so create a `struct` called "person"
+Before a table can be configured and instantiated, we need to define a struct that represents the data structure of the address book. Since it is an address book, the table will contain people, so create a `struct` called "person." For this exercise, create the struct` called "person" in the `private` section of the "addressbook" class.
```
Comment (iamveritas) [inline docs/02_getting-started/03_smart-contract-development/04_data-persistence.md:44]: I would go for this

Before a table can be configured and instantiated, we need to define a struct that represents the data structure of the address book. The address book will contain people, therefore for this exercise, create the struct` called "person" in the `private` section of the "addressbook" class.

**Comment by iamveritas** [comment]
I think you'll not be successful to merge into master
it has to be either develop or release/2.0.x branches

**Comment by rusty-block-one** [comment]
Pull request targeted to wrong branch

**Comment by rusty-block-one** [comment]
Thank you for the review and feedback regarding merge into master.  I have
cancelled this pull request to master and generated a new pull request to
release/2.0x.  Would you mind reviewing the pull request.  It can be found
at https://github.com/EOSIO/welcome/pull/347


On Mon, Nov 9, 2020 at 2:42 PM ovi <notifications@github.com> wrote:

> ------------------------------
> *[External Email: Use caution with links and attachments]*
> ------------------------------
>
> I think you'll not be successful to merge into master
> it has to be either develop or release/2.0.x branches
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_EOSIO_welcome_pull_346-23issuecomment-2D724266460&d=DwMCaQ&c=sSVGO1PAPMGyxGLZsRfH-A&r=pkfTcH0ayH99kU33lwT_PrBAd0cPNSZG8hx9C_FLq3E&m=rUXGbxHL3wun93h0eexTz93BbbGXepx8bopkj7mvPcY&s=PIl8fnrj0aa-31NsgFIg7M7NJVTHM4OuSFgWkkIBZNg&e=>,
> or unsubscribe
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_ARQVQ7OV3UK7HVYMBZWFV33SPBH3RANCNFSM4TPYB2EQ&d=DwMCaQ&c=sSVGO1PAPMGyxGLZsRfH-A&r=pkfTcH0ayH99kU33lwT_PrBAd0cPNSZG8hx9C_FLq3E&m=rUXGbxHL3wun93h0eexTz93BbbGXepx8bopkj7mvPcY&s=63YEMRbynkkGZ0TxoiirJZqd4QALH_aqfrILIfm3adQ&e=>
> .
>


-- 

*Russell Fleming*
*Contractor - Accolite*
russell.fleming-ext@block.one

*The information contained in this email is confidential together with any
information contained in any attachments to this email. You should not
disclose, give or copy the information to anyone other than the person(s)
who are named or referenced above. The law may also protect the information
as a legally privileged document and/or as a copyright work. If you have
received this email and you are not the person named or referenced above,
please let the sender know and delete the email from your computer (and, if
applicable, your computer system(s) and server(s)). For information about
how we process personal data and monitor communications please see our
privacy policy <https://block.one/privacy-policy/> and for terms of use
please see our terms of use here. <https://block.one/terms-of-use/>*

**Pull request closed without merging**